doc/example-config-apache.md

```diff
@@ -15,18 +15,18 @@ virtual host.
 ## Apache configuration sniplet
 
 ```
-<Location /webrtc>
-    ProxyPass http://127.0.0.1:8080/webrtc
-    ProxyPassReverse /webrtc
-</Location>
-
-<Location /webrtc/ws>
-    ProxyPass ws://127.0.0.1:8080/webrtc/ws
-</Location>
-
-ProxyVia On
-ProxyPreserveHost On
-RequestHeader set X-Forwarded-Proto 'https' env=HTTPS
+    <Location /webrtc>
```

Comment:
Well the indentation is intentional as those blocks are meant to be copy/pasted into existing blocks which require indentation to be clean. If will change everything to use tab-indentation.